conda-recipe/run_test.bat

```diff
@@ -32,10 +32,10 @@ if not defined PYTHON (
 
 
 "%PYTHON%" -c "import dpnp; print(dpnp.__version__)"
-if errorlevel 1 exit 1
+if errorlevel neq 0 exit 1
```

Comment:
It seems doesn't work properly.
All tests passed there:
https://cje-fm-owrp-prod04.devtools.intel.com/satg-dap-intelpython/job/intel-packages/job/dpnp/job/dev-windows-py3.11/job/test/1493/cloudbees-pipeline-explorer/
but job returned error status.